.github/workflows/php.yml

```diff
@@ -69,9 +69,24 @@ jobs:
       - name: Generate Schema
         run: composer run framework:schema:dump
 
+      - name: "Restore result cache"
+        uses: actions/cache/restore@v4
+        with:
+          path: var/cache/phpstan
+          key: "phpstan-result-cache-${{ github.run_id }}"
+          restore-keys: |
+            phpstan-result-cache-
+
       - name: PHPStan
         run: composer run phpstan -- --error-format=github --no-progress
 
+      - name: "Save result cache"
+        uses: actions/cache/save@v4
+        if: always()
```

Comment:
maybe it is enough to only save result cache on nightly?

Reply:
hope, this works out :see_no_evil: I remember we had some troubles with old caches :thinking:

Reply:
Let's try it, the cache is never shared with other branches, branches only get the cache from the same branch or default branch

Reply:
Upload only took 3 seconds with are size of 12 MB. I think it's ok to always do that